config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
```

Comment:
- [ ] I wasn't presented with the registration page at the beginning. I think this page should be presented to the user before any action that's `http://localhost:3000/users/sign_in` should be the default page. When I hit on to http://localhost:3000/ a page showing the list of all users was open which I think shouldn't be the case.